_posts/guides/2021-12-21-guide_gamemode-info-bhop.md

```diff
@@ -19,14 +19,32 @@ toc_sticky: true
 
 Bhop, short for "***B***unny ***Hop***", is a very popular gamemode found in many games, such as CS:GO, TF2, and others.
 
```

Comment:
Should add another paragraph above the other one explaining what "bunny hopping" means (i.e., jumping as soon as you hit the ground). that'll make explaining scrollhop/autohop in the next section make more sense